web/packages/teleport/src/Assist/contexts/messages.tsx

```diff
@@ -271,6 +271,23 @@ function findIntersection<T>(elems: T[][]): T[] {
   return [...elems.map(e => new Set(e)).reduce(intersectSets)];
 }
 
+export async function generateTitle(messageContent: string): Promise<string> {
+  const res = (await api.post(`/v1/webapi/assistant/title/summary`, {
+    message: messageContent,
+  })) as { title: string };
+
+  return res.title;
```

Comment:
This is rather cumbersome to read. WDYT about splitting it to get the await on one line and either specify the type in the binding with `const name: type` or cast it during the return.

Reply:
```typescript
  const res: { title: string } = await api.post(`/v1/webapi/assistant/title/summary`, {
    message: messageContent,
  });
```

would work here